toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -667,39 +693,38 @@ func calculateTotalPackages(packages []string, installRoot string) (totalPackage
 		// Search for the list of packages to be installed,
 		// it will be prefixed with a line "Installing:" and will
 		// end with an empty line.
-		inPackageList := false
 		for _, line := range splitStdout {
-			const (
-				packageListPrefix    = "Installing:"
-				packageNameDelimiter = " "
-			)
+			matches := listedPackageRegex.FindStringSubmatch(line)
```

Comment:
As above, can we unify with the code in `ClonedRepoContents()`, maybe in `tdnf.go`?

Reply:
Sure, I'll move listedPackagesRegex to tdnf.go. Are you suggesting to move all other related variables as well?

Reply:
The ones that make sense, there is a lot of overlap here in both the regex and functions.